scss/core/base.scss

```diff
@@ -404,15 +404,15 @@ html.no-touchevents.js-focus-visible {
       }
   
       &:focus:not(.focus-visible) {
-        box-shadow: none !important;
+        box-shadow: none;
```

Comment:
I don't like that we've got CSS styles in here that don't use any of the variables.

If our core CSS files had the following:

```css
html.no-touchevents.js-focus-visible .focus-outline-active,
html.no-touchevents.js-focus-visible .focus-outline {
  outline: none;
}

html.no-touchevents.js-focus-visible .focus-outline-active:focus:not(.focus-visible),
html.no-touchevents.js-focus-visible .focus-outline-active:not(:focus-visible),
html.no-touchevents.js-focus-visible .focus-outline:focus:not(.focus-visible),
html.no-touchevents.js-focus-visible .focus-outline:not(:focus-visible) {
  box-shadow: none;
}
```

...could we then clean up this section of the file a bit?

Reply:
This hasn't been responded to.

If the Fliplet core CSS file for all the pages contain the mentioned CSS above, will this work? I don't like that the CSS declaration doesn't make use any of the variables, which means this is the wrong place to be defining them.

Reply:
@tonytlwu Yes, it should work if we add the above CSS globally in the Fliplet studio. I though that you wonted just to group selectors.

Reply:
@YaroslavOvdii This has been added to development now. See below.

![image](https://user-images.githubusercontent.com/290733/114418344-55bb0e80-9baa-11eb-8790-396ab42a79ed.png)

Could you remove the static properties (ones that don't use any variables) and re-test your solution?

Reply:
@tonytlwu Yes, this solution is working. [Screencast](https://share.getcloudapp.com/yAuDlNko).

Reply:
@YaroslavOvdii The solution shows that the box shadows are working again, which is great. However, this changes how outlines are applied. Can you also confirm that this doesn't break the implementation of focus outlines across different device types?

Reply:
@tonytlwu Yes, accessibility is working as well on all resolutions. [Screencast](https://share.getcloudapp.com/mXup6eEe)

Reply:
@YaroslavOvdii Looks good 👍🏻

Reply:
@tonytlwu Removed unneeded styles from the theme.